File: baekjoon/bronze/level2/test_2577/java/Main.java
package baekjoon.bronze.level2.test_2577.java;

import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        int a = Integer.parseInt(br.readLine());
        int b = Integer.parseInt(br.readLine());
        int c = Integer.parseInt(br.readLine());
        String str = String.valueOf(a * b * c);

        for (int i = 0; i < 10; i++) {
            int l = str.length();
            str = str.replace(String.valueOf(i), "");
            bw.write(String.valueOf(l - str.length()));
            bw.newLine();
        }
        bw.close();
    }
}

File: baekjoon/bronze/level2/test_2577/kotlin/Main.kt
package baekjoon.bronze.level2.test_2577.kotlin

fun main() {
    val a = readln().toInt()
    val b = readln().toInt()
    val c = readln().toInt()

    val sum = (a * b * c).toString()
    val arr = IntArray(10)
    for (c in sum.toCharArray()) {
        arr[c.digitToInt()]++
    }

    for (i in 0..9) println(arr[i])
}
